alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -879,24 +879,27 @@ impl Canvas {
     /// `self.width` and an attached rectangle to form a "╭" using a given `stroke_size` in the
     /// bottom-right quadrant of the `Canvas` coordinate system.
     fn draw_rounded_corner(&mut self, stroke_size: usize) {
-        let r = 0.5 * (self.width.min(self.height) + stroke_size) as f32;
+        let r = (self.width.min(self.height) + stroke_size) as f32 / 2.;
```

Comment:
Still would be nice to have not one character names for variables. Same applies for `d_bias` and such.